src/utils/common.js

```diff
@@ -595,3 +596,13 @@ export const formatBigFiatAmount = (value: Value, fiatCurrency: string) => {
 export const getEnsName = (username: string) => `${username}${getEnsPrefix()}`;
 
 export const extractUsernameFromEnsName = (ensName: string) => ensName.replace(getEnsPrefix(), '');
+
+export const addressAsKey = (address: string): string => {
+  if (!address) return '';
+  return address.toLowerCase();
+};
+
+export const valueForAddress = <RecordValue>(
+  record: Record<RecordValue>,
+  address: string,
+): ?RecordValue => record[addressAsKey(address)];
```

Comment:
Consider allowing for nil record to avoid need for `?? {}` in various call sites.

```suggestion
export const valueForAddress = <Value>(
  record: ?Record<Value>,
  address: string,
): ?Value => record?.[addressAsKey(address)];
```

Reply:
`Value` is reserved for imported type in the above, added changes for nullish.